File: Java/src/main/java/com/gildedrose/categories/ConjuredManaCake.java
package com.gildedrose.categories;

import com.gildedrose.*;

public class ConjuredManaCake extends DefaultCategory {

    @Override
    public String getName() {
        return "Conjured Mana Cake";
    }

    @Override
    public void UpdateQuality(Item item) {
        super.UpdateQuality(item);
        super.UpdateQuality(item);
    }
}

File: csharpcore/Categories/ConjuredManaCake.cs
namespace csharpcore.Categories
{
    public class ConjuredManaCake : DefaultCategory
    {
        public override string Name => "Conjured Mana Cake";

        public override void UpdateQuality(Item item)
        {
            base.UpdateQuality(item);
            base.UpdateQuality(item);
        }
    }
}